### ISSUE TITLE
ability to toggle cell_y_sort

### ISSUE THREAD

**Issue by girng** [vnen/godot-tiled-importer#98]
**Plugin version**
2.2, 3.1 godot

**Issue description**
i have around 5+ layers, and they all get imported with `cell_y_sort` to true. for example, i don't want this to happen for a ground tile.
so i add:
https://i.gyazo.com/9a63f86356e894c388a5713c6ac41b60.png
`cell_y_sort` to the tile property as **false**, but it's not working am not sure

**Comment by vnen** [comment]
You can use a post-import script for that.

TBH I don't know if I'll ever add this. TileMap has a lot of properties and using custom properties for this is quite arbitrary. That applies to other nodes that can be created on import as well (Sprite, StaticBody, etc.)

**Comment by girng** [comment]
good idea. forgot about that import script. i just had re-check and disable all y sort options cause performance was killing me, didn't realize it was enabled on all my ground tilelayers lol

ty
